Servers/UI/OJS.Servers.Ui.Models/Contests/ContestFiltersAndSortingRequestModel.cs

```diff
@@ -23,9 +23,12 @@ public class ContestFiltersRequestModel : IMapExplicitly
 
     public int? ItemsPerPage { get; set; }
 
+    [BindProperty(Name = "sorttype")]
+    public string SortType { get; set; }
+
     public void RegisterMappings(IProfileExpression configuration)
         => configuration
```

Comment:
Add a map to convert from `string` to `enum`. See comment in `ContestFiltersAndSortingServiceModel.cs`